statistics/fmsketch.go

```diff
@@ -201,3 +205,18 @@ func (s *FMSketch) MemoryUsage() (sum int64) {
 	sum = int64(16 + 9*len(s.hashset))
 	return
 }
+
+func (s *FMSketch) reset() {
+	maps.Clear(s.hashset)
+	s.mask = 0
+	s.maxSize = 0
+}
+
+// Destroy resets the FMSketch and puts it back to the pool.
+func (s *FMSketch) Destroy() {
```

Comment:
I suggest we name it something like `DestroyAndPutToPool`.
Because it's meaningless to explicitly "destroy", the main point is that we want to put it back in the pool. Also, I think it's important that other people are aware that we'll reuse this struct when calling this method.

Reply:
I have updated.